### ISSUE TITLE
Im receiving a Net::ReadTimeout: timed out while waiting for more data

### ISSUE THREAD

**Issue by Hamdan85** [zacstewart/tengai#12]
I'm trying to use horizons api but keep receiving Net::ReadTimeout: timed out while waiting for more data.
What am I doing wrong?

**Comment by zacstewart** [comment]
Oh boy. It could be anything. I haven't looked at this code in so long, and it's kind of brittle because it relied on a NASA telnet service to retrieve the data. If they changed that service, it could have broken it.

**Comment by Hamdan85** [comment]
Are you willing to fix it? Im looking for ephemeris in angles... im getting
XYZ positions... so, by now, I'll need to study some astronomy in order to
calculate them myself.

**Comment by zacstewart** [comment]
I don't think so. I don't use or have time to maintain this little gem anymore. I would recommend getting the data straight from HORIZONS itself via the bulk ephemeras API.

**Comment by Hamdan85** [comment]
Where did you saw ephemeras API via rest? I'm looking for it for sometime and didnt found!

**Comment by zacstewart** [comment]
https://ssd.jpl.nasa.gov/horizons.cgi <https://ssd.jpl.nasa.gov/horizons.cgi>
